tests/test-cases/Group1-Docker-Commands/1-04-Docker-Create.robot

```diff
@@ -208,3 +208,35 @@ Create disables VC destroy
     Should Be Equal As Integers  ${rc}  0
     Run Keyword If  '%{HOST_TYPE}' == 'VC'  Should Contain  ${output}  Destroy_Task
     Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Should Not Contain  ${output}  Destroy_Task
+
+Parallel creates with same container name
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull ${busybox}
+    Should Be Equal As Integers  ${rc}  0
+
+    ${pid1}=  Start Process  docker %{VCH-PARAMS} create --name testDuplicates ${busybox}  shell=True
```

Comment:
I would suggest to create a unique random suite variable instead of this fixed and repetitive `testDuplicates` to avoid typo issues and other issues involving concurrent test executions later.

Reply:
Good point - fixed.